src/components/v2/Icon/icons/fee.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M11.996 21.998C17.5166 21.998 21.992 17.5213 21.992 11.999C21.992 6.4767 17.5166 2 11.996 2C6.47535 2 2 6.4767 2 11.999C2 17.5213 6.47535 21.998 11.996 21.998Z" fill="currentColor"/>
+<path d="M9.366 11.648C8.666 11.648 8.09667 11.4333 7.658 11.004C7.21933 10.5653 7 10.0147 7 9.352C7 8.68 7.21933 8.12 7.658 7.672C8.09667 7.224 8.666 7 9.366 7C10.0753 7 10.6493 7.224 11.088 7.672C11.5267 8.12 11.746 8.68 11.746 9.352C11.746 10.0147 11.5267 10.5653 11.088 11.004C10.6493 11.4333 10.0753 11.648 9.366 11.648ZM9.506 16.478H8.456L14.42 7.14H15.484L9.506 16.478ZM16.212 15.988C15.7733 16.4267 15.204 16.646 14.504 16.646C13.804 16.646 13.2347 16.4267 12.796 15.988C12.3573 15.5493 12.138 14.9987 12.138 14.336C12.138 13.664 12.3573 13.104 12.796 12.656C13.2347 12.208 13.804 11.984 14.504 11.984C15.204 11.984 15.7733 12.208 16.212 12.656C16.66 13.104 16.884 13.664 16.884 14.336C16.884 14.9987 16.66 15.5493 16.212 15.988ZM9.366 10.556C9.69267 10.556 9.95867 10.4487 10.164 10.234C10.3693 10.01 10.472 9.716 10.472 9.352C10.472 8.96933 10.3693 8.666 10.164 8.442C9.95867 8.218 9.69267 8.106 9.366 8.106C9.03933 8.106 8.77333 8.218 8.568 8.442C8.36267 8.666 8.26 8.96933 8.26 9.352C8.26 9.70667 8.36267 9.996 8.568 10.22C8.78267 10.444 9.04867 10.556 9.366 10.556ZM13.706 15.232C13.9113 15.4467 14.1773 15.554 14.504 15.554C14.8307 15.554 15.0967 15.4467 15.302 15.232C15.5167 15.008 15.624 14.7093 15.624 14.336C15.624 13.9533 15.5167 13.65 15.302 13.426C15.0967 13.202 14.8307 13.09 14.504 13.09C14.1773 13.09 13.9113 13.202 13.706 13.426C13.5007 13.65 13.398 13.9533 13.398 14.336C13.398 14.7093 13.5007 15.008 13.706 15.232Z" fill="white"/>
```

Comment:
shouldn't we use currentColor here as well?

Reply:
In this instance we can't, otherwise the "%" would be blended into the background:
<img width="186" alt="Screenshot 2022-02-23 at 21 02 38" src="https://user-images.githubusercontent.com/44675210/155407936-dede53f8-6232-4c76-9977-25255a11d063.png">